packages/next/src/client/components/router-reducer/create-initial-router-state.ts

```diff
@@ -132,7 +135,7 @@ export function createInitialRouterState({
         // add a way to split a single Flight stream into static and dynamic
         // parts. But in the meantime we should at least make this work for
         // fully static pages.
-        staleTime: -1,
+        staleTime: prerendered ? STATIC_STALETIME_MS : -1,
```

Comment:
I believe this is functionally equivalent to the logic [here](https://github.com/vercel/next.js/blob/c450cd83d1ad34e7906582573584956f401feb2a/packages/next/src/client/components/router-reducer/prefetch-cache-utils.ts#L353-L356).